server/cmd/mmctl/commands/plugin_e2e_test.go

```diff
@@ -379,3 +379,134 @@ func (s *MmctlE2ETestSuite) TestPluginDeleteCmd() {
 		s.Require().Len(plugins.Inactive, 1)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestPluginDisableCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	pluginID := "testplugin"
+	pluginURL := filepath.Join(server.GetPackagePath(), "tests", "testplugin.tar.gz")
+	nonExistentPluginID := "nonExistentPluginID"
+
+	// Enable plugin uploads
+	s.th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.PluginSettings.Enable = true
+		*cfg.PluginSettings.EnableUploads = true
+	})
+
+	// Install plugin first
+	err := pluginAddCmdF(s.th.SystemAdminClient, &cobra.Command{}, []string{pluginURL})
+	s.Require().Nil(err)
+	defer removePlugin(s, pluginID)
+
+	s.RunForSystemAdminAndLocal("Successful disable plugin", func(c client.Client) {
+		printer.Clean()
+
+		appErr := s.th.App.EnablePlugin(pluginID)
+		s.Require().Nil(appErr)
+
+		plugins, appErr := s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 1)
+		s.Require().Len(plugins.Inactive, 0)
+
+		cmd := &cobra.Command{}
+		err := pluginDisableCmdF(c, cmd, []string{pluginID})
+		s.Require().Nil(err)
+
+		plugins, appErr = s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 0)
+		s.Require().Len(plugins.Inactive, 1)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], "Disabled plugin: "+pluginID)
+	})
+
+	s.Run("error for disable plugin", func() {
```

Comment:
nit: I'd make a reference to permissions on the title of the test, something like:

```suggestion
	s.Run("error for disable plugin due insufficient permissions", func() {
```

Reply:
Resolved the comments.